xpublish/plugins/included/dataset_info.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
```

Comment:
question about file naming. Do we need the `/included/` part of the file path for the default plugins?

Reply:
I was wavering between `/default/` and `/included/` for the plugins that come with xpublish. I did want to keep them in the `/plugins/` directory, but I wanted to differentiate them from the plugin machinery. I initially had `/included_plugins/` or similar at the same level as `/plugins/` but that seemed like overkill and more confusing.

I'm also thinking that as part of documenting things, I'll probably make a few demo plugins that won't be automatically loaded via entrypoints, but can show off the different parts of the functionality (for example an dataset plugin that provides the xarray tutorial datasets), so I was thinking those would end up in `xpublish/plugins/demo/`

Reply:
I think this is a bit overkill at this point but I'm fine with it.